public/api/conf/1.0/schemas/state-pension.json

```diff
@@ -0,0 +1,180 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Regular Response",
+  "type": "object",
+  "properties": {
+    "exclusionReasons": {
+      "type": "array",
+      "items": {
+      "type": "string",
+        "oneOf": [
+          {
+            "enum": [ "Abroad" ],
+            "description": "Male who has an overseas address (outside United Kingdom and Isle of Man) and retires before 06/10/2018. Unable to automate currently, manual intervention required."
```

Comment:
Male who has an address outside the United Kingdom and Isle of Man, and retires before 6 October 2018. Unable to automate currently, manual intervention required.